### PR TITLE
added functionality for viewing Doom Thresholds

### PR DESCRIPTION
**Pull request by dscarpac**
changes made to the doomcounter and doominplay scripts

### CODE REVIEW

File: src/core/DoomCounter.ttslua [hunk 1]
```diff
@@ -87,3 +157,28 @@ function toggleOptions()
     self.UI.hide("Options")
   end
 end
+
+-- searches an area and optionally filters the result
+function searchArea(origin, size, filter)
```
[the remark this answers is not in the document]
Comment (dscarpac) [reply src/core/DoomCounter.ttslua:162]: Yeah, I agree. It's duplicated in playermat/playmat.

[the remark this answers is not in the document]
Comment (Chr1Z93) [reply src/core/DoomCounter.ttslua:162]: Good idea. We could make a shared utility file with such functions (we use them in several object's script)

File: src/core/DoomCounter.ttslua [hunk 2]
```diff
@@ -72,6 +91,57 @@ function startReset()
   end
 end
 
+-- get Doom Threshold from top card of Agenda deck
+function getDoomThreshold()
+ 
+  local origin = { -2.72, 1.6, 0.37 } -- this needs to be edited to the actual coordinates of the agenda deck
```
[the remark this answers is not in the document]
Comment (Chr1Z93) [reply src/core/DoomCounter.ttslua:97]: Will be resolved in separate PR

File: src/core/DoomCounter.ttslua (hunk 1 above)
[the remark this answers is not in the document]
Comment (Chr1Z93) [reply src/core/DoomCounter.ttslua:162]: Will be resolved in separate PR

**Review by Chr1Z93 — APPROVED** [review]
Feature complete - will be slightly reworked with shared utility file in separate PR

**Pull request merged**